vinculacion/vinculacion_main.py

```
@@ -0,0 +1,150 @@
+            
+    async def perform_new_ups_linking(self) -> Dict:
+        """
+        Only matches UPs.
+        
+        Returns:
+            Dict: Results of incremental linking
+        """
+        return #TODO: Implement incremental linking
+        check_date = self.config.get_linking_target_date() #93 days back from today|    
+        
+        print(f"\n🔄 STARTING INCREMENTAL UOF-UP LINKING")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)